### ISSUE TITLE
Indentation: false positive for try child on the same line

### ISSUE THREAD

**Issue by pbludov** [checkstyle/checkstyle#5685]
https://checkstyle.org/checks/misc/indentation.html#Indentation

`/var/tmp $ javac TestClass.java`:
`/var/tmp $ cat TestClass.java`:

```
class TestClass {
    void method() {
        try { return; // Line 3
        }
        catch (Exception e) { return; // Line 5
        }
    }
}
```

`/var/tmp $ cat config.xml`:

```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="Indentation"/>
  </module>
</module>
```

`/var/tmp $ java -Duser.language=en -Duser.country=US -jar checkstyle-8.8-all.jar -c config.xml TestClass.java`:

```
Starting audit...
[ERROR] /var/tmp/TestClass.java:3: 'try' child has incorrect indentation level 8, expected level should be 12. [Indentation]
[ERROR] /var/tmp/TestClass.java:5: 'catch' child has incorrect indentation level 8, expected level should be 12. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```

There should be no violations.

**Comment by harsh-kukreja** [comment]
Hey @romani according to the conventions we should not allow users to write a code like this, isn't it?

```
class TestClass {
    void method() {
        try { return; // Line 3
        }
        catch (Exception e) { return; // Line 5
        }
    }
}
```

Instead, Users should write a code
```
class TestClass {
    void method() {
        try { return; }
        catch (Exception e) { return; }
    }
}
```
OR
```
class TestClass {
    void method() {
        try {
            return;
        }
        catch (Exception e) { 
            return; 
        }
    }
}
```

**Comment by rnveach** [comment]
Indentation is not strictly about how could should be written but how it should be formatted, or specifically idented on each line. Users may have their code written like this for a specific reason, like they prefer single lined try/catch when it is 1 statement, but they ran out of space due to max line length and had to split it up into 2 different lines.

To me, based on the user's config, the violations are wrong as the code looks properly formatted as it is right now.

**Comment by Carbon14-48** [comment]
i'm on this !

**Comment by romani** [comment]
Fix is merged
